vendor/revisions.json

```diff
@@ -1,5 +1,5 @@
 {
-  "v16": ["16.3", "b39f316137fdd29e2da15d2af2fdd1cfd18163be"],
+  "v16": ["16.3", "2e372c392105e709cca8d9224c9c0a1fa5bd3aff"],
```

Comment:
Please
- add a link to the corresponding Postgres PR to this PR description
- reference the issue in both PRs -- https://github.com/neondatabase/cloud/issues/15361

Cannot review the code part of the PR, so added Heikki as well